docs/enterprise-starter.mdx

```diff
@@ -0,0 +1,67 @@
+
+The Enterprise Starter plan offers a multi-tenant Sourcegraph designed for small teams and individuals that require the same core features of a traditional Sourcegraph instance but with a simplified management experience. This plan provides a fully managed version of Sourcegraph (AI + code search + omnibox, with privately indexed code) through a self-serve flow.
+
+## Team seats
+
+The Enterprise Starter plan is priced at **$19 per month per seat**. You can add or remove team members at any time. Existing Cody Pro users can also sign up for the Enterprise Starter by paying $19 per seat. However, their Cody Pro subscription will neither be upgraded nor canceled. Instead, they will have two live subscriptions.
+
+## Enterprise Starter team roles
+
+The Enterprise Starter plan includes the following team-level roles:
+
+- **Admin**: Has full access to the workspace, including the ability to manage users, billing, and settings
+- **Member**: Can access repositories in the workspace and can experience the supported features
+
+## Billing
+
+Workspaces on the Enterprise Starter plan are billed monthly based on the number of team seats purchased. In case of overdue or failing payment, there is a grace period during which the workspace admins will receive a daily notification to complete the transaction.
+
+## Features supported
+
+The Enterprise Starter plan supports a variety of AI and search-based features like:
+
+|             **AI features**              |        **Code Search**         |     **Deployment**     |                      **Management**                       |        **Support**        |
+| ---------------------------------------- | ------------------------------ | ---------------------- | --------------------------------------------------------- | ------------------------- |
+| Unlimited autocomplete and chat messages | Indexed Code Search            | Multi-tenant cloud     | Simplified admin experience with UI-based repo-management | Support with limited SLAs |
+| Powerful LLM models for chat             | Indexed Symbol Search          | Private workspace      | User management                                           | -                         |
+| Omnibox intent detection                 | Searched based code-navigation | Privately indexed code | GitHub code host integration                              | -                         |
+| Cody integration                         | -                              | -                      | -                                                         | -                         |
+
+
+<Callout type="note">We are actively working on bringing more features to Enterprise Starter. Stay tuned for updates!</Callout>
+
+## Limits
+
+Sourcegraph Enterprise Starter offers the following limits:
+
+- Max 50 users per workspace
+- Max 100 repos
+- 5 GB of storage
```

Comment:
Maybe instead say "Starts with 5 gb storage"

Reply:
Is there an upper limit?

Reply:
Each workspace starts with 5 gigs of storage and then gets 1 additional gig per user added up to 10 gigs. 

We will likely revisit these limits in the future but this is our starting point

Reply:
@mmanela just wanted to confirm that it is an additional 1GB per user, I heard elsewhere that it's an extra 500mb per user, up to the 10GB upper limit.

Reply:
It is additional 1 gig per user up to 10gigs